packages/css/src/components/card-grid/card-grid.scss

```diff
@@ -0,0 +1,10 @@
+.dso-card-grid {
+  display: grid;
+  gap: $grid-gutter-width;
+  grid-template-columns: repeat(4, 1fr);
+  grid-auto-rows: 1fr;
+
+  .dso-card {
+    box-shadow: 0 4px 8px 0 rgb(0 0 0 / 20%);
```

Comment:
zou 't ns een keer tijd zijn om de waarde voor box-shadow centraal te borgen?
![image](https://user-images.githubusercontent.com/2689559/147093857-de4b89df-8cf5-431a-ab4e-ea1c69054b81.png)

Reply:
Verder heb ik de maten nagelopen en deze lijken te kloppen met het ontwerp, voor wat betreft de paddings en margins (eigenlijk: de `gap`).

In het ontwerp echter staat een hoogte van 120px. Da's natuurlijk gek, want is helemaal afhankelijk van de vulling van de cards. Door `grid-auto-rows: 1fr;` blijven/worden ze wel allemaal netjes even hoog.

Reply:
De verschillende box-shadow's pakken we in #963 op